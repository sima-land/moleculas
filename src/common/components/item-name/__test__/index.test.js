import React from 'react';
import ItemName from '../index';
import { mount } from 'enzyme';

describe('<ItemName />', () => {
  it('should render without props', () => {
    const wrapper = mount(<ItemName />);
    expect(wrapper).toMatchSnapshot();
  });
  it('should render with props', () => {
    const wrapper = mount(<ItemName name='Foo Bar' href='#' shrinkable />);
    expect(wrapper).toMatchSnapshot();
  });
});
