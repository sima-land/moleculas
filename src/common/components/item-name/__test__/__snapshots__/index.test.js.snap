// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ItemName /> should render with props 1`] = `
<ItemName
  href="#"
  name="Foo Bar"
  shrinkable={true}
>
  <div
    className="wrapper"
  >
    <Link
      href="#"
    >
      <a
        className="link link-blue link-can-be-hovered"
        href="#"
        target="_self"
      >
        <Memo(Text)>
          <span
            className=""
          >
            <span
              className="name"
            >
              Foo Bar
            </span>
          </span>
        </Memo(Text)>
      </a>
    </Link>
  </div>
</ItemName>
`;

exports[`<ItemName /> should render without props 1`] = `
<ItemName>
  <div
    className="wrapper"
  >
    <Link>
      <span
        className="link link-blue link-can-be-hovered"
      >
        <Memo(Text)>
          <span
            className=""
          >
            <span
              className="name"
            />
          </span>
        </Memo(Text)>
      </span>
    </Link>
  </div>
</ItemName>
`;
